aldryn_events/tests/frontend/unit/test.cl.events.js

```diff
@@ -3,20 +3,50 @@
  * @copyright: http://www.divio.ch
  */
 
-'use strict';
+(function () {
+    'use strict';
 
-// #####################################################################################################################
-// #NAMESPACES#
-/* global Cl, describe, it, expect */
+    // ########################################################################
+    /* global Cl, describe, it, expect */
 
-// #####################################################################################################################
-// #UNIT TEST#
-describe('cl.events.js', function () {
-    it('Cl namespace is available', function () {
-        expect(Cl).toBeDefined();
-    });
+    // ########################################################################
+    describe('cl.events.js', function () {
+        beforeEach(function () {
+            fixture.setBase('frontend/fixtures');
+            this.markup = fixture.load('calendar.html');
+        });
+
+        afterEach(function(){
+            fixture.cleanup();
+        });
+
+        it('Cl namespace is available', function () {
+            expect(Cl).toBeDefined();
+        });
+
+        it('has a public method calendar', function () {
+            expect(Cl.events.calendar).toBeDefined();
+        });
 
-    it('has a public methode calendar', function () {
-        expect(Cl.events.calendar).toBeDefined();
+        it('Cl.events.init() returns undefined', function () {
+            expect(Cl.events.init()).toEqual(undefined);
+        });
+
+        it('runs calendar() in Cl.events.init()', function () {
+            spyOn(Cl.events, 'calendar');
+            Cl.events.init();
+
+            // validate that calendar was called inside Cl.events.init()
+            expect(Cl.events.calendar).toHaveBeenCalled();
+            expect(Cl.events.calendar.calls.count()).toEqual(1);
+        });
+
+        it('Cl.events._handler returns false if direction is not specified', function () {
+            expect(Cl.events._handler.call(
+                $('.js-trigger')[0],
```

Comment:
we never really use `$.fn.get` sadly, in this case it's the same thing, `[0]` being a tiniest bit faster, I wouldn't change it